### PR TITLE
added rough screenshot tool

### PR DESCRIPTION
**Pull request by sorja**
example output

screenshot_FIN

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -229,6 +229,7 @@
     "passport-local": "^1.0.0",
     "pg": "^8.7.3",
     "pg-promise": "^10.10.2",
+    "puppeteer-cluster": "^0.23.0",
```
Comment (minotogna) [inline package.json]: Should  this go under dev dependencies ?
Doesn't it get imported from puppeteer ?

File: src/tools/screenshotWebpage.js [hunk 2]
```diff
@@ -0,0 +1,296 @@
+const puppeteer = require('puppeteer')
```
Comment (minotogna) [inline src/tools/screenshotWebpage.js]: Better to use typescript ?

File: src/tools/screenshotWebpage.js [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+  'X16',
+  'X17',
+  'X18',
+  'X19',
+  'X20',
+  'YEM',
+  'ZAF',
+  'ZMB',
+  'ZWE',
+]
```
Comment (minotogna) [inline src/tools/screenshotWebpage.js]: Better to fetch country isos ?

File: src/tools/screenshotWebpage.js (hunk 2 above)
Comment (sorja) [reply src/tools/screenshotWebpage.js]: sure, this was just a quick tool to quickly get the request done but i can change it to TS

**Comment by sorja** [comment]
@minotogna let me know if this should be converted to TypeScript
from my side we can leave it as it is in JavaScript, as it is rarely used and is more a reference if we need to run this tool again

**Review by minotogna — APPROVED** [review]

**Review by minotogna — CHANGES_REQUESTED** [review]
Why not ts ?

**Comment by minotogna** [comment]
> @minotogna let me know if this should be converted to TypeScript
> 
> from my side we can leave it as it is in JavaScript, as it is rarely used and is more a reference if we need to run this tool again

@sorja it's always preferred typescript

**Comment by sorja** [comment]
> Why not ts ?

to avoid compiling, but i can change to ts!
will do at some point, i guess this is not extremely high priority :)

**Review by minotogna — APPROVED** [review]

**Pull request merged**